install-javafx.html

```diff
@@ -63,11 +63,12 @@ <h2>Run HelloWorld using JavaFX <span class="JFX_MAJOR">11</span></h2>
     </div>
 </div>
 
-<p>
-    <b>Note:</b> Additional modules are required for extended functionality.
+<div class="alert alert-warning">
+    <strong>Note:</strong>
+    Additional modules are required for extended functionality.
     For example, if your application is using <kbd>FXML</kbd>, you will need to
     add the <kbd>javafx.fxml</kbd> module, as shown below:
-</p>
+</div>
```

Comment:
I'd extend the `<div>` until the next paragraph ends,
```
since it is transitively required by the <kbd>javafx.controls</kbd> module).
</div>
```
Since that will include code, the code tag should use:
`<pre class="alert"><code class="alert">`

Reply:
Sorry I'm not sure that I understand what you mean?
The `note` box should start at line 66 and end line 98. So we have the the to tabsections in the note?

Reply:
From 66 to 94. The tab section is a problem as it is too dark over the warning background. Adding `class="alert"` removes the dark background for the code (not for the tab). See how it looks like.

Reply:
I update this section. Can you have a look on it?
Should we add a 'break line' after the bold 'Note:'? Normally in the document the sections with 'Note:' have a 'break line'.